pipelines/sync-hac-frontend-configs/tasks.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     This task updates redhat-appstudio-qe/frontend-configs with 
     content from internal gitlab repo insights-platform/frontend-configs.
   steps:
-    - image: quay.io/devfile/base-developer-image:latest
+    - image: quay.io/devfile/base-developer-image@sha256:07f34105a8708123bb7e73d47d737aa256a2b43f878ad9ad2b8e06fa680c0c3b
```

Comment:
I'm not sure if we need to pin the dev image that's not used in e2e-tests. @rhopp wdyt?

Reply:
yep. I agree. No reason for pinning this one ;-)

Reply:
Changed back to latest.